picoquic/sender.c

```diff
@@ -319,10 +319,21 @@ uint32_t picoquic_create_packet_header_12(
         /* Create a short packet -- using 32 bit sequence numbers for now */
         uint8_t K = (packet_type == picoquic_packet_1rtt_protected_phi0) ? 0 : 0x40;
         const uint8_t C = 0x30;
+	uint8_t VEC = (cnx->VEC );
```

Comment:
Same remark about version dependency. We cannot implement VEC with draft-11. We have to either write some version dependent test, or decide to just forget about draft 11 compatibility. But if we do that, I need to make a snapshot of the code and archive it before we merge this PR.